COM-807 PMTCT_FO - Number of HIV-exposed infants with a documented outcome by 18 months - HIV-final status unknown

tmvumbi2 opened the pull request:
Jira ticket: https://jembiprojects.jira.com/browse/COM-807

tmvumbi2 committed:
COM-807 Add testing indicator 6c
Implements the testing indicator 6c

lauravignoli commented on metadata/reportssql/testing_report_functions.sql:
```diff
@@ -601,6 +636,33 @@ BEGIN
+    JOIN appointment_service aps ON aps.appointment_service_id = pa.appointment_service_id AND aps.voided = 0
+    WHERE pa.voided = 0
+        AND pa.patient_id = p_patientId
+        AND pa.status = "Completed"
```
@tmvumbi2 Do we need to check that status is Completed?

Checked with Richard, and he said we don't need to check that the appointment was Completed. We need to check that there was an appointment scheduled within the reporting period and an HIV related visit after the scheduled date within the reporting period..

tmvumbi2 committed:
COM-807 Fix testing indicator 6c
- removes checking that the appointment status is "completed": an appointment is considered attended when there exist a HIV related visit after the appointment's date
- adds a condition to Part 1 to ensure that there is no open-elis PCR test order for which no result has been captured yet
- fixes the age's condition
